ui/droppable.js

```diff
@@ -33,6 +33,7 @@ $.widget( "ui.droppable", {
 		accept: "*",
 		activeClass: false,
 		addClasses: true,
+		classes: {},
 		greedy: false,
 		hoverClass: false,
```

Comment:
One thing i'm not sure about here. This current controls if ANY class is added by default there is none. So to make this work with classes, we would either always add a `ui-droppable-active` class, and then the value would be configurable, to add additional classes. Other wise we would switch this to keep both of the current options, but have them control if a key from the classes option ( that would have no value by default ) is added or not. 

@scottgonzalez @mikesherov thoughts?